spec/index.html

```diff
@@ -503,10 +503,10 @@ <h3>Blank nodes</h3>
 	<li> [I+A](x)=I(x) when x is a <a>name</a>; </li>
 	<li> [I+A](x)=A(x) when x is a blank node; </li>
 	<li> [I+A](x)= RE( [I+A](x.s), [I+A](x.p), [I+A](x.o) ) when x is a triple term, where x.s, x.p, and x.o are the first, second, and third components of x, respectively; </li>
-	<li> [I+A](x)=true when x is a triple and [I+A](p) is in IP and the pair <[I+A](s),[I+A](o)> is in IEXT([I+A](p));
+	<li> [I+A](x)=true when x is a triple and [I+A](p) is in IP and the pair <[I+A](s),[I+A](o)> is in IEXT([I+A](p));</li>
 	<li> [I+A](x)=false when x is a triple, otherwise;
-	<li> [I+A](x)=false if x is an RDF graph and [I+A](x')=false for some triple x' in x;
-        <li> [I+A](x)=true if x an RDF graph, otherwise
+	<li> [I+A](x)=false if x is an RDF graph and [I+A](x')=false for some triple x' in x;</li>
+    <li> [I+A](x)=true if x an RDF graph, otherwise</li>
```

Comment:
```suggestion
	<li> [I+A](x)=false if x is an RDF graph and [I+A](x')=false for some triple x' in x;</li>
	<li> [I+A](x)=true if x an RDF graph, otherwise</li>
```